### PR TITLE
Bugfix/um 856 hip rangelo

### PR DESCRIPTION
**Pull request by kab163**

### CODE REVIEW

File: benchmarks/vendor_allocator_benchmarks.cpp [hunk 1]
```diff
@@ -21,16 +21,26 @@
...
 
-static const int RangeLow{1<<10}; //1kB
+/*
+ * Note: HIP runs need their own RangeLow since hipMalloc 
+ * makes allocations aligned along 4k pages. HIP UM and 
+ * PINNED memory currently have a bug which may require
+ * further modification of the ranges used.
+ */
+#if defined(UMPIRE_ENABLE_HIP)
+  static const int RangeLow{1<<12}; //1kB
```
Comment (davidbeckingsale) [inline benchmarks/vendor_allocator_benchmarks.cpp:31]: Comment `//1kB` is wrong here.

Comment (kab163) [reply benchmarks/vendor_allocator_benchmarks.cpp:31]: ah good catch

**Review by mcfadden8 — COMMENTED** [review]
I wonder if we should simply change the rangelo to 4k for all of the device types in order to keep the ranges used the same when showing comparisons?

**Comment by kab163** [comment]
> I wonder if we should simply change the rangelo to 4k for all of the device types in order to keep the ranges used the same when showing comparisons?

Yea good point. I think the key question is whether or not any valuable info is gained by doing an additional range value for CUDA. The cudaMalloc has the ability to do 1024 byte allocations until exhausting max memory, while hipMalloc can't - but is that necessarily important to know? (Keeping it all the same rangelo will definitely make it look nicer hehe)

**Review by davidbeckingsale — APPROVED** [review]

**Review by mcfadden8 — APPROVED** [review]

**Pull request merged**